docs/reference/drawing.rst

```diff
@@ -69,8 +72,9 @@ Pegasus Graph Functions
 .. autosummary::
    :toctree: generated/
 
-   draw_pegasus
+     draw_pegasus
```

Comment:
should these all be at the same indentation level?

Reply:
Yes, I had tabs mixed in with spaces. I will fix them.